src/components/Home/Home.css

```diff
@@ -0,0 +1,111 @@
+.imageScore {
+  width: 50%;
+}
+
+.imageScore img {
+  width: 100%;
+}
+
+.games {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  align-items: center;
+  justify-content: center;
+  color: lightgray;
+}
+
+.title {
+  color: white;
+}
+
+.arrow-right {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.arrow-btn {
+  background: none;
+  border: none;
+}
+
+.arrow {
+  font-size: 1.5rem;
+  color: white;
+  position: absolute;
+  top: 1rem;
+  right: 1rem;
+}
+
+.game {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 0.5rem;
+  height: 25vh;
+  position: relative;
+}
+
+.game h2 {
+  color: lightgray;
+  font-size: 1rem;
+}
```

Comment:
- [ ] Kindly make sure this text is aligned at the left as in template

The template | your design
---|---
<img width="153" alt="image" src="https://user-images.githubusercontent.com/65068771/234230266-f1df73d2-7684-4515-87ee-ac49f79e53a2.png"> | <img width="334" alt="image" src="https://user-images.githubusercontent.com/65068771/234230479-0e09a3f6-797f-4f84-a6cb-bd56c2dfc477.png">

- [ ] Also make sure you reduce the font size it should look like the template. check the difference in the above above screenshots